crates/ruff/src/importer/mod.rs

```diff
@@ -149,31 +148,58 @@ impl<'a> Importer<'a> {
     /// the name on which the `lru_cache` symbol would be made available (`"functools.lru_cache"`).
     fn import_symbol(
         &self,
-        module: &str,
-        member: &str,
+        symbol: &ImportableSymbol,
         at: TextSize,
         semantic_model: &SemanticModel,
     ) -> Result<(Edit, String), ResolutionError> {
```

Comment:
Should this now return a `ImportableSymbol` instead of a `string` or some other enum denoting the used import style? Or is this not possible, because of aliases?